test/test_fcl_capsule_capsule.cpp

```diff
@@ -126,6 +125,7 @@ BOOST_AUTO_TEST_CASE(distance_capsulecapsule_transformZ)
 
 BOOST_AUTO_TEST_CASE(distance_capsulecapsule_transformZ2)
 {
+  const FCL_REAL Pi = std::acos(FCL_REAL(-1.0));
```

Comment:
I was trying to fix what appeared to be incorrect use of floating point type. Using macros for constants means they will be untyped and might not convert properly to FCL_REAL. I took a look at boost::pi and can't figure out what type it is! I don't think it is a good idea to have the digits of pi included here -- there must be a better way! What would you suggest as the proper way to get a "pi" of type FCL_REAL?